sdk/python/feast/feature.py

```diff
@@ -62,3 +63,86 @@ def from_proto(cls, feature_proto: FeatureProto):
         feature.update_shape_type(feature_proto)
         feature.update_domain_info(feature_proto)
         return feature
+
+
+class FeatureRef:
+    """ Feature Reference represents a reference to a specific feature.  """
+
+    def __init__(self, name: str, feature_set_name: str = None):
```

Comment:
I know you are matching the proto field names here, but eventually I think we should rename these to `feature` and `feature_set`. Now might be the time.

Reply:
Changed `FeatureReference`'s `feature_set_name` field to `feature_set`